src/components/Common/UserColumns.tsx

```diff
@@ -0,0 +1,31 @@
+import { UserModel } from "../Users/models";
+
+export type userChildProps = {
+  userData: UserModel;
+  username: string;
+  refetchUserData?: () => void;
+};
+
+export default function userColumns(
```

Comment:
We can make this a react component and args could be passed as props right?

```suggestion
export default function UserColumns(
```

Reply:
Yep, can do.